### ISSUE TITLE
Loose order of some attributes (attributes-order)

### ISSUE THREAD

**Issue by rainfore** [vuejs/eslint-plugin-vue#515]
**Tell us about your environment**

* **ESLint Version: 4.19.1**
* **eslint-plugin-vue Version: 4.5.0**
* **Node Version: 8.9.4**

**Please show your full configuration:**

``` js
{
    extends: 'plugin:vue/recommended',
    parser: 'vue-eslint-parser',
    plugins: ['vue'],
    rules: {
        'vue/attributes-order': [2],
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

``` vue
<template>
<div>
    <u-button ref="button" color="primary" href="https://vuejs.org/" disabled style="width: 100%" @click="onClick">Vue</u-button>
    <u-button ref="button" color="primary" :href="href" :disabled="disabled" :style="styleObject" @click="onClick">Vue</u-button>
</div>
</template>
```

**What did you expect to happen?**

``` vue
<template>
<div>
    <u-button ref="button" color="primary" href="https://vuejs.org/" disabled style="width: 100%" @click="onClick">Vue</u-button>
    <u-button ref="button" color="primary" :href="href" :disabled="disabled" :style="styleObject" @click="onClick">Vue</u-button>
</div>
</template>
```

**What actually happened? Please include the actual, raw output from ESLint.**
``` vue
<template>
<div>
    <u-button ref="button" color="primary" href="https://vuejs.org/" disabled style="width: 100%" @click="onClick">Vue</u-button>
    <u-button ref="button" :href="href" :disabled="disabled" :style="styleObject" color="primary" @click="onClick">Vue</u-button>
</div>
</template>
```

It often happens in developing that I exchange same attributes from normal to binding. I want a consistent order by their name but eslint fixer will force to change it. But I don't want just to close this rule.
So is there any way to ignore the order of BINDING and OTHER_ATTR and keep other types?

Rule config like this:
``` js
rules: {
    'vue/attributes-order': [2, { order: [
        'DEFINITION',
        'LIST_RENDERING',
        'CONDITIONALS',
        'RENDER_MODIFIERS',
        'GLOBAL',
        'UNIQUE',
        ['BINDING', 'OTHER_ATTR'],
        'EVENTS',
        'CONTENT',
    ] }],
}
```

**Comment by michalsnik** [comment]
Hey @rainfore It has been fixed in #421 and waits for the next major release :)

**Comment by rainfore** [comment]
Thanks! I think my issue is duplicated with #410
